content/en/docs/tasks/configure-pod-container/configure-service-account.md

```diff
@@ -199,9 +202,9 @@ The content of `token` is elided here.
     ```
 
 - Verify it has been created.
-   ```shell
-   kubectl get secrets myregistrykey
-   ```
+    ```shell
+    kubectl get secrets myregistrykey
+    ```
```

Comment:
This additional indentation is antipattern. Please change it back.

Reply:
Thank,I will change it back.